### PR TITLE
[#12588] Improve test code coverage of CopySessionModalComponent

### PR DESCRIPTION
**Pull request by Astrid641**
Part of #12588

**Outline of Solution**

 
Add test for the 'copy' method and improve the test for 'select' method from partial covered to all covered for CopySessionModalComponent.

### CODE REVIEW

File: src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts [hunk 1]
```diff
@@ -81,15 +83,6 @@ describe('CopySessionModalComponent', () => {
...
 
-  it('should snap with some session and courses candidates', () => {
-    component.newFeedbackSessionName = feedbackSessionToCopy.feedbackSessionName;
-    component.courseCandidates = [courseSessionIn, courseCopyTo];
-    component.sessionToCopyCourseId = courseSessionIn.courseId;
-    fixture.detectChanges();
-
-    expect(fixture).toMatchSnapshot();
-  });
-
```
Comment (cedricongjh) [inline src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts:92]: let's keep this snapshot test here

**Review by cedricongjh — CHANGES_REQUESTED** [review]
thank you for the PR, just one comment!

File: src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts (hunk 1 above)
Comment (Astrid641) [reply src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts:92]: Thanks for the comment and reply, I actually struggled dealing with running errors with this part. I did not edit any part of “this should snap with some session and course candidates‘, however, it met this error when I kept the snapshot test. Thanks for your support in advance.
image

Comment (cedricongjh) [reply src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts:92]: hey @Astrid641 it appears that `copy-session-modal.component.spec.ts.snap` was changed, do revert the change and the test should pass

Comment (Astrid641) [reply src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts]: Thanks for your reply! I have reverted the change and now all checks have passed, hope all goes well now.

**Comment by Astrid641** [comment]
Changes made

**Review by cedricongjh — APPROVED** [review]
LGTM, thank you for the PR!

**Review by weiquu — APPROVED** [review]
LGTM

**Pull request merged**